tracer/src/Datadog.Trace/ClrProfiler/CallTarget/CallTargetInvoker.cs

```diff
@@ -528,7 +528,7 @@ static CallTargetInvoker()
             if (IntegrationOptions<TIntegration, TTarget>.IsIntegrationEnabled)
             {
                 IntegrationOptions<TIntegration, TTarget>.RecordTelemetry();
-                return EndMethodHandler<TIntegration, TTarget>.Invoke(instance, exception, in state);
+                EndMethodHandler<TIntegration, TTarget>.Invoke(instance, exception, in state);
```

Comment:
We don't return here anymore?

Reply:
this is the `void` case so `CallTargetReturn` is just an empty struct. In case of returning something it will be `CallTargetReturn.GetDefault();` and that's is the exact line after the `if`.

Reply:
I see. Thanks for clarifying.